models/demos/roberta/demo/demo.py

```diff
@@ -250,44 +255,45 @@ def run_roberta_question_and_answering_inference_squad_v2(
             i += 1
         eval_score = squad_metric.compute(predictions=pred_labels, references=true_labels)
         cpu_eval_score = squad_metric.compute(predictions=cpu_pred_labels, references=true_labels)
-        logger.info(f"\tTT_Eval: exact: {eval_score['exact']} --  F1: {eval_score['f1']}")
+        logger.info(f"TT_Eval: exact: {eval_score['exact']} --  F1: {eval_score['f1']}")
 
 
-@pytest.mark.parametrize("model_name", ["deepset/roberta-large-squad2"])
-@pytest.mark.parametrize("bert", [ttnn_bert, ttnn_optimized_bert])
-def test_demo(
-    input_path,
-    model_name,
-    bert,
-    device,
-    use_program_cache,
-):
+@pytest.mark.parametrize(
+    "model_name, input_loc",
+    ((["deepset/roberta-large-squad2", "models/demos/roberta/demo/input_data.json"]),),
+)
+@pytest.mark.parametrize(
+    ("bert", "batch_size", "sequence_size"),
+    ((ttnn_optimized_bert, 8, 384),),
+)
+def test_demo(device, use_program_cache, model_name, input_loc, bert, batch_size, sequence_size):
```

Comment:
Most other models will do some sort of check against the reference model responses, or a known set of answers. We don't have any asserts for correctness in this test, which makes it mostly pointless.

Reply:
The goal of this test was to secure that demo executes. It is for functionality of demo, not accuracy.
Now that there is a test with squad data this is not necessary.